src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml

```diff
@@ -1,321 +1,350 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nice one that you managed to export it from the Flowable online editor. Would be nice to document this somewhere in our developer docs I think. For another PR maybe. :-)

Reply:
Added `cmmn.md`